airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -341,9 +341,10 @@ public AirbyteCatalog discover(JsonNode config) throws Exception {
               .map(f -> Field.of(f.getColumnName(), JdbcUtils.getType(f.getColumnType())))
               .distinct()
               .collect(Collectors.toList());
-          final String streamName = JdbcUtils.getFullyQualifiedTableName(t.getSchemaName(), t.getName());
-          final List<String> primaryKeys = tablePrimaryKeys.getOrDefault(streamName, Collections.emptyList());
-          return new TableInfo(streamName, t.getSchemaName(), fields, primaryKeys);
+          final String fullyQualifiedTableName = JdbcUtils.getFullyQualifiedTableName(t.getSchemaName(), t.getName());
+          final List<String> primaryKeys = fullyQualifiedTableNameToPrimaryKeys.getOrDefault(fullyQualifiedTableName, Collections.emptyList());
+
+          return new TableInfo(t.getName(), t.getSchemaName(), fields, primaryKeys);
```

Comment:
Isn't it possibly confusing that it is not consistent on the ordering of tableName/schemaName in the function arguments?

`public TableInfoInternal(String schemaName, String tableName, List<ColumnInfo> fields) `
vs
`public TableInfo(String name, String schemaName, List<Field> fields, List<String> primaryKeys)`

Usually it's maybe more natural to refer schemaName then tablename?

Reply:
good point. will change.

Reply:
on second look, I feel the `catalogHelpers`'s param should be changed to `namespace` to make it clear. what do you think?

Reply:
yes sure!